### ISSUE TITLE
Invariant Violation: onlyChild must be passed a children with exactly one child.

### ISSUE THREAD

**Issue by ronag** [js-next/react-style#92]
When doing server side rendering I get ```Invariant Violation: onlyChild must be passed a children with exactly one child.```

I'm using 0.4.0.

This gives me the error:

```

      <DocumentTitle title='Media Asset Manager'>
        <div styles={this.styles.background}>
          <RouteHandler />
        </div>
      </DocumentTitle>
```

This doesn't:

```
      <DocumentTitle title='Media Asset Manager'>
        <div>
          <RouteHandler />
        </div>
      </DocumentTitle>
```

```
Error: Invariant Violation: onlyChild must be passed a children with exactly one child.
    at invariant (/Users/ronagy/GitHub/foo/node_modules/react/lib/invariant.js:42:15)
    at Object.onlyChild [as only] (/Users/ronagy/GitHub/foo/node_modules/react/lib/onlyChild.js:29:44)
    at [object Object].React.createClass.render (/Users/ronagy/GitHub/foo/node_modules/react-document-title/node_modules/react-side-effect/index.js:52:31)
    at [object Object].ReactCompositeComponentMixin._renderValidatedComponent (/Users/ronagy/GitHub/foo/node_modules/react/lib/ReactCompositeComponent.js:1260:34)
    at [object Object].wrapper [as _renderValidatedComponent] (/Users/ronagy/GitHub/foo/node_modules/react/lib/ReactPerf.js:50:21)
    at [object Object].ReactCompositeComponentMixin.mountComponent (/Users/ronagy/GitHub/foo/node_modules/react/lib/ReactCompositeComponent.js:802:14)
    at [object Object].wrapper [as mountComponent] (/Users/ronagy/GitHub/foo/node_modules/react/lib/ReactPerf.js:50:21)
    at [object Object].ReactCompositeComponentMixin.mountComponent (/Users/ronagy/GitHub/foo/node_modules/react/lib/ReactCompositeComponent.js:808:44)
    at [object Object].wrapper [as mountComponent] (/Users/ronagy/GitHub/foo/node_modules/react/lib/ReactPerf.js:50:21)
    at [object Object].ReactCompositeComponentMixin.mountComponent (/Users/ronagy/GitHub/foo/node_modules/react/lib/ReactCompositeComponent.js:808:44)
    at [object Object].wrapper [as mountComponent] (/Users/ronagy/GitHub/foo/node_modules/react/lib/ReactPerf.js:50:21)
    at /Users/ronagy/GitHub/foo/node_modules/react/lib/ReactServerRendering.js:40:38
    at ReactServerRenderingTransaction.Mixin.perform (/Users/ronagy/GitHub/foo/node_modules/react/lib/Transaction.js:134:20)
    at Object.renderToString (/Users/ronagy/GitHub/foo/node_modules/react/lib/ReactServerRendering.js:38:24)
    at getPageHtml (/Users/ronagy/GitHub/foo/src/server/render.js:39:40)
    at Function.<anonymous> (/Users/ronagy/GitHub/foo/src/server/render.js:28:18)```

**Comment by ronag** [comment]
An array gets passed into ``ònlyChild```not sure why.

**Comment by ronag** [comment]
See #93

**Comment by SanderSpies** [comment]
Thanks for fixing this, will merge soonish.

**Comment by SanderSpies** [comment]
Done - published 0.4.1 for this :+1:
